test/functional/feature_pak.py

```diff
@@ -63,6 +63,8 @@ def run_test(self):
 
         assert_raises_rpc_error(-8, "bip32_counter must be between 0 and 1,000,000,000, inclusive.", self.nodes[1].initpegoutwallet, xpub, 1000000001)
 
+        assert_raises_rpc_error(-8, "bitcoin_descriptor is not a valid descriptor string", self.nodes[1].initpegoutwallet, "[deadbeef/44h/0h/0h]")
```

Comment:
this error is pretty obscure(does this mean it's fixed? I can't really tell), can you instead insert it into one of the example descriptors that doesn't have it, and make sure it passes checks?

Reply:
hmm yeah sorry, this doesn't make sense

Reply:
fixed, btw